main/src/ca/uwaterloo/flix/runtime/evaluator/SymbolicEvaluator.scala

```diff
@@ -749,26 +749,6 @@ object SymbolicEvaluator {
           case v => throw InternalCompilerException(s"Type Error: Unexpected value: '$v'.")
         }
 
-      /**
```

Comment:
In this file you can just throw internal runtime exception for the array expressions.